work_queue/src/bindings/python3/PyTask/PyTask.py

```diff
@@ -0,0 +1,107 @@
+import work_queue as wq
+import dill
+import os
+import sys
+import tempfile
+import uuid
+import traceback
+
+
+class PyTask(object):
+	
+	p_tasks = []
+
+	def __init__(self, func, *args):
+		self._id = str(uuid.uuid4())
+		self._tmpdir = 'tmp/'
+		self._task = self.python_function_command(func, *args)
+		PyTask.p_tasks.append(self)
+		
+	
+	def task(self):
+		return self._task
+	
+	def python_function_command(self, func, *args):
+			
+		func_file = os.path.join(self._tmpdir, 'function_{}.p'.format(self._id))
+		args_file = os.path.join(self._tmpdir, 'args_{}.p'.format(self._id))
+		out_file = os.path.join(self._tmpdir, 'out_{}.p'.format(self._id))
+		package_file = os.path.join(self._tmpdir, 'dep.yml')
+		tar_file = os.path.join(self._tmpdir, 'venv.tar.gz')
+		pp_run = '/afs/crc.nd.edu/user/b/bslydelg/cctools/python_environments/src/python_package_run'		
```

Comment:
This hardcoded path won't work for anyone except you.  :)
How about `shutil.which("python_package_run")` or similar.